libs/top_level_metrics.py

```diff
@@ -0,0 +1,129 @@
+import numpy as np
+import pandas as pd
+
+from covidactnow.datapublic.common_fields import CommonFields
+from libs.datasets import combined_datasets
+from libs.datasets.timeseries import TimeseriesDataset
+
+
+def calculate_top_level_metrics_for_fips(fips: str):
+    timeseries = combined_datasets.load_us_timeseries_dataset()
+    latest = combined_datasets.load_us_latest_dataset()
+
+    fips_timeseries = timeseries.get_subset(fips=fips)
+    fips_record = latest.get_record_for_fips(fips)
+
+    # not sure of return type for now, could be a dictionary, or maybe it would be more effective
+    # as a pandas dataframe with a column for each metric.
+    return calculate_top_level_metrics_for_timeseries(fips_timeseries, fips_record)
+
+
+def calculate_top_level_metrics_for_timeseries(timeseries: TimeseriesDataset, latest: dict):
+    # Making sure that the timeseries object passed in is only for one fips.
+    assert len(timeseries.all_fips) == 1
+    fips = timeseries.all_fips[0]
+    population = latest.population
+    neg_tests_cumulative = timeseries.data[CommonFields.NEGATIVE_TESTS]
+    neg_tests_daily = neg_tests_cumulative.diff()
+
+    cases_cumulative = timeseries.data[CommonFields.CASES]
+    pos_tests_cumulative = timeseries.data[CommonFields.CASES]
+    date = timeseries.data[CommonFields.DATE]
+    pos_tests_daily = pos_tests_cumulative.diff()
+    cases_daily = cases_cumulative.diff()
+
+    case_density = calculate_case_density(cases=cases_daily, population=population)
+    test_positivity = calculate_test_positivity(
+        pos_cases=pos_tests_daily, neg_tests=neg_tests_daily
+    )
+
+    top_level_metrics_data = {
+        "caseDensity": case_density,
```

Comment:
I think it would be good to make these constants.  It's a bit of a trick that we're using to say "the key is the same name as the output of the API definition".  Definitely possible that we could rename it.  One suggestion I have is to make a simple class (or enum, similar to [CommonFields](https://github.com/covid-projections/covid-data-public/blob/master/covidactnow/datapublic/common_fields.py#L25)) that has the names of the metrics columns as keys.  

that way this could be `MetricsFields.CASE_DENSITY` and this is a bit less tightly coupled to the API output name